CHANGES.rst

```diff
@@ -134,6 +134,8 @@ astropy.modeling
 - Add a ``separability_matrix`` function which returns the correlation matrix
   of inputs and outputs. [#7803]
 
+- Fixed a bug in ``JointFitter``. [#7984]
```

Comment:
A little vague - what about:
```suggestion
- Fixed compatibility of ``JointFitter`` with the latest version of Numpy. [#7984]
```

Reply:
Too bad I looked at this after I looked at my email and pushed a change. I can't try the suggested change now.